src/env/test/mod.rs

```diff
@@ -98,6 +98,16 @@ impl TestEnv {
     pub fn rng(&mut self) -> &mut TestRng256 {
         &mut self.rng
     }
+
+    pub fn set_enterprise_attestation(&mut self) {
```

Comment:
For a `set_*` function, I'd expect an argument that is written into some property. This is more of an "enable" I think?